### PR TITLE
chore: update redpacket contract

### PR DESCRIPTION
**Pull request by zhouhanseng**

### CODE REVIEW

File: packages/maskbook/src/plugins/RedPacket/types.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export interface RedPacketAvailability {
     total: string
     claimed: string
     expired: boolean
-    ifclaimed: boolean
+    claimed_amounts: number
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/RedPacket/types.ts:36]: Will it break the current code? Does this object published as a payload or stored in the database?

File: packages/maskbook/src/plugins/RedPacket/constants.ts [hunk 2]
```diff
@@ -7,8 +7,8 @@ export const RedPacketPluginID = 'com.maskbook.red_packet'
 export const RED_PACKET_CONSTANTS = {
     HAPPY_RED_PACKET_ADDRESS: {
         [ChainId.Mainnet]: '0x26760783c12181efa3c435aee4ae686c53bdddbb',
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/RedPacket/constants.ts:9]: mainnet doesnt update, does it mean mainnet still using the old contract?

File: packages/maskbook/src/plugins/RedPacket/types.ts (hunk 1 above)
Comment (zhouhanseng) [reply packages/maskbook/src/plugins/RedPacket/types.ts:36]: Neither of them, actually `ifclaimed` has not been used before. And this object will be calculated every time when load the red packet to decide the status.

File: packages/maskbook/src/plugins/RedPacket/constants.ts (hunk 2 above)
Comment (zhouhanseng) [reply packages/maskbook/src/plugins/RedPacket/constants.ts:9]: ye

**Review by guanbinrui — APPROVED** [review]

**Comment by yisiliu** [comment]
I don't get it - why would you @zhouhanseng merge this into the master without deploying the new contract?

**Comment by zhouhanseng** [comment]
Lack of experience. This PR should not merge into master, otherwise it's possible to be released.

**Pull request merged**